python/cugraph/structure/renumber.py

```diff
@@ -167,4 +167,10 @@ def renumber_from_cudf(_df, source_cols_names, dest_cols_names):
     _dst_ids = _tmp_df_dst.merge(
         _tmp_df, on=_vals, how='left').drop(_vals).sort_values(by='index')
 
-    return _src_ids['id'], _dst_ids['id'], _tmp_df
+    _s_id = cudf.Series(_src_ids['id']).reset_index(drop=True)
```

Comment:
We could change the line154 similar to this, i.e, _tmp_df = _tmp_df.drop_duplicates().reset_index(drop=True) since it is faster